contrib/opencensus-ext-azure/examples/metrics/sum.py

```diff
@@ -41,13 +41,14 @@ def main():
     exporter = metrics_exporter.new_metrics_exporter(export_interval=5)
```

Comment:
I suggest that we give a reasonable exporter interval comparable to the ingestion and processing latency.